modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/constants/liferaysDomainList.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+export const LiferayDomainsList: Array<string> = [
+	'bp.liferay.com',
+	'connect.liferay.com',
+	'dev.liferay.com',
+	'gs.liferay.com',
+	'in.liferay.com',
+	'liferay.com',
+	'liferay.io',
+	'testray.io',
+];
```

Comment:
1. Avoid naming `list`
2. camelCase for variable names
3. Not necessary to type `Array<string>` because the implicit type is already set
4. We can add `as const` to add each value of array as type, but this is optional.

```suggestion
export const liferayDomains = [
	'bp.liferay.com',
	'connect.liferay.com',
	'dev.liferay.com',
	'gs.liferay.com',
	'in.liferay.com',
	'liferay.com',
	'liferay.io',
	'testray.io',
] as const;
```

Reply:
Thanks <3